bin/prepare_gff_for_conversion.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+# Copyright 2023-2024 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import re
+
+
+def main(infile, outfile):
+    with open(infile, "r") as file_in, open(outfile, "w") as file_out:
+        fasta_flag = False
+        for line in file_in:
+            if line.startswith("##FASTA"):
+                fasta_flag = True
+                file_out.write(line)
+            elif line.startswith("#"):
+                file_out.write(line)
+            elif fasta_flag:
+                file_out.write(line)
+            else:
+                new_line = modify_line(line)
+                file_out.write(new_line + "\n")
+
+
+def modify_line(line):
+    columns = line.strip().split("\t")
+    if columns[2] in ["region", "tRNA"]:
+        return line.strip()
+
+    # Parse the 9th field into key-value pairs
+    attributes_dict = dict(
+        re.split(r"(?<!\\)=", item) for item in re.split(r"(?<!\\);", columns[8])
+    )
+    # Move all GO terms into Dbxref and deduplicate if needed
+    attributes_dict = collect_go_terms(attributes_dict)
+
+    # Move Pfam, Rfam and InterPro into Dbxref
+    terms_to_move = ["pfam", "rfam", "interpro"]
+    attributes_dict = move_to_dbxref(attributes_dict, terms_to_move)
+
+    attributes_to_move_to_note = {
```

Comment:
Why do you change the names?

Reply:
That's the convention for these names, they are defined in Dbxref. If not part of Dbxref, they need to be lowercase.